src/client/game.cpp

```diff
@@ -3731,14 +3732,37 @@ bool Game::nodePlacement(const ItemDefinition &selected_def,
 
 		// Don't place node when player would be inside new node
 		// NOTE: This is to be eventually implemented by a mod as client-side Lua
```

Comment:
You can probably remove this comment, I doubt this is ever going to happen.

Reply:
sounds like a plan